source/includes/_amp_snippet.md

```diff
@@ -2,9 +2,20 @@
 
 AMP snippet is a piece of code that allows to render Tipser collections on an AMP page.
 
+Due to the limitations of AMP format, AMP snippet is limited in functionality compared to Tipser Widget.
+
+Significant limitations:
+- only a colection of products can be displayed, no way to display a single product (if this is your need, then you can create a Tipser collection with just a single product)
```

Comment:
no way to display a single product (if this is your need, then you can create a Tipser collection with just a single product)  => if you need to display only one product,  please create a Tipser collection with just one product.

"no way" seems too final :)

Reply:
Good point, thanks :)